content/posts/August-2021.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "August 2021"
+date: 2021-08-06
+draft: false
+weight: 72
+---
+
+## Week 31 (August 2nd - August 6th)
+
+- `sync_release` in API allows specifying a suffix for the newly created branch in a dist-git repo fork,
+  so that there can be more open update PRs for the same dist-git branch at the same time.
+  ([packit#1326](https://github.com/packit/packit/pull/1326))
```

Comment:
you found it even I had forgot to add the `has-release-notes` label, well done :-)

Reply:
I still went through the PRs manually since the template is there for just a few days. But it's really nice that I can use the text without much thinking..;)